### PR TITLE
dev(pds-ember): update blueprints

### PR DESCRIPTION
**Pull request by CITguy**
quick update of the blueprints to reduce the amount of repetitive work for components

### CODE REVIEW

File: packages/pds-ember/blueprints/pds-component/files/__root__/components/__emberPrefix__/__name__/docs.mdx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Props } from '@storybook/addon-docs/blocks';
+import { ArgsTable } from '@storybook/addon-docs/blocks';
```
Comment (CITguy) [inline packages/pds-ember/blueprints/pds-component/files/__root__/components/__emberPrefix__/__name__/docs.mdx:1]: Storybook 6

File: packages/pds-ember/blueprints/pds-component/files/__root__/components/__emberPrefix__/__name__/index.hbs [hunk 2]
```diff
@@ -1,3 +1,10 @@
-<div class="<%= cssClassName %>" data-test-root>
+<div
+  class="<%= cssClassName %>"
+  ...attributes
+  data-test-root
```
Comment (meirish) [inline packages/pds-ember/blueprints/pds-component/files/__root__/components/__emberPrefix__/__name__/index.hbs:4]: I still think you'll want unique data-keys per component here, but no reason to change it now.

**Review by meirish — APPROVED** [review]
Nice update here 👍

File: packages/pds-ember/blueprints/pds-component/files/__root__/components/__emberPrefix__/__name__/index.hbs (hunk 2 above)
Comment (CITguy) [reply packages/pds-ember/blueprints/pds-component/files/__root__/components/__emberPrefix__/__name__/index.hbs:4]: This is a good starting point and I've been changing this for each component anyway, but I didn't want to spend cycles getting it perfect.

**Pull request merged**